htdocs/index.php

```diff
@@ -1,3 +1,3 @@
 <?php
-
+	// Here, we become infinite.
```

Comment:
This is what I've been living for this entire time and now I can do it forever. Thanks PHPainfree!

Reply:
@y04nqt right now

![raw](https://user-images.githubusercontent.com/30528226/113312648-1ea64c80-92d0-11eb-898a-a13e06c2c35a.gif)

Reply:
![youreright](https://user-images.githubusercontent.com/10656995/113313077-85c40100-92d0-11eb-9d5e-1f5f52d4eaa4.gif)